content/bdk-cli/regtest.md

```diff
@@ -25,7 +25,7 @@ electrs -vv --timestamp --db-dir /tmp/electrs-db --electrum-rpc-addr="127.0.0.1:
 
 on macOS you should change the cookie-file to `$HOME/Library/Application Support/Bitcoin/regtest/.cookie`.
 
-This will start the Electrum server on port 50001. You can then add the `-n regtest -s localhost:50001` to the `bdk-cli` commands to switch to the local regtest.
+This will start the Electrum server on port 50001. You can then add the `-n regtest -s 127.0.0.1:50001` to the `bdk-cli` commands to switch to the local regtest.
```

Comment:
looks good, can you also change the `localhost` to `127.0.0.1` on line 52?

Reply:
Thanks for comment. You're right. I missed it because I make PR only from github GUI.
If there are any rules for PR, e.g. squashing commit, I can make it. Please tell me.

Reply:
Squash is fine, thanks!

Reply:
I squashed commits.